signals/CS/MWC-USD.yaml

```diff
@@ -0,0 +1,7 @@
+sources:
+- source_id: coingecko
+  id: mimblewimblecoin
+- source_id: coinmarketcap
+  id: '5031'
+- source_id: cryptocompare
+  id: MWC-USD
```

Comment:
not available in free, default config of bothan

Reply:
the registry is only here to provide the source mapping, whether or not the individual implements the source or not is out of this scope

Reply:
ok understandable